doc/modules/linear_model.rst

```diff
@@ -175,13 +175,12 @@ This method has the same order of complexity as
 .. between these
 
 
-Setting the regularization parameter: generalized Cross-Validation
+Setting the regularization parameter: leave-one-out Cross-Validation
 ------------------------------------------------------------------
```

Comment:
`/home/circleci/project/doc/modules/linear_model.rst:179: WARNING: Title underline too short.'`

Reply:
How do I resolve this?

Reply:
The dash line length should match the title length (a bit silly, but it is enforced in the doc tests).